content/events/cshl-biology-of-genomes-2023.md

```diff
@@ -0,0 +1,37 @@
+---
+author: "AnVIL"
+conference: "CSHL BoG"
+description: "The 2023 meeting will address DNA sequence variation and its role in molecular evolution, population genetics and complex diseases, comparative genomics, large-scale studies of gene and protein expression, and genomic approaches to ecological systems."
+eventType: "Conference"
+featured: true
+location: "Cold Spring harbor Laboratory, NY, USA"
```

Comment:
`Harbour` <-- upper case

Reply:
Going to make it Harbor because that's what it is in American(and on the issue), but should have caught the capitalization!